build.assets/Dockerfile-grpcbox

```diff
@@ -8,6 +8,13 @@ RUN apt-get update && \
     && \
   rm -rf /var/lib/apt/lists/*
 
+# Add the dev user.
+ARG UID
+ARG GID
+RUN groupadd dev --gid=$GID -o && \
+    useradd dev --uid=$UID --gid=$GID --create-home --shell=/bin/sh && \
+    mkdir -p -m0700 /var/lib/teleport && chown -R dev /var/lib/teleport
```

Comment:
We don't care about `/var/lib/teleport` here, this shouldn't be part of this image.